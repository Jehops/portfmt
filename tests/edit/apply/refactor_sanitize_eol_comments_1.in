MAKE_ARGS=	Q= # verbose builds
