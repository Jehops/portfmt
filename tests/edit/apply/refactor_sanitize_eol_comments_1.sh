${PORTEDIT} apply refactor.sanitize-eol-comments refactor_sanitize_eol_comments_1.in | \
	diff -L refactor_sanitize_eol_comments_1.expected -L refactor_sanitize_eol_comments_1.actual \
		-u refactor_sanitize_eol_comments_1.expected -
