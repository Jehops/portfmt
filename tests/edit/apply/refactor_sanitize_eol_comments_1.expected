# verbose builds
MAKE_ARGS=	Q=
